packages/rulesets/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/openapi-validator-rulesets",
-  "version": "2.0.10",
+  "version": "2.0.11",
```

```suggestion
  "version": "2.1.0",
```

Let's start following semver.  Increment the minor when adding new rules/features, and increment patch if the only change is fixing bugs in existing rules.

---

Sure will do, thanks